.github/workflows/container-tests.yml

```diff
@@ -17,72 +17,67 @@ jobs:
             os_test: "fedora"
             context: "Fedora"
             compose: "CentOS-7"
```

Comment:
Do we want to test Fedora on Centos7 compose?

Reply:
Yeah, we do it. We test Fedora on CEntOS7.